source/Components/AvalonDock/VisualTreeHelperExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿/************************************************************************
+   AvalonDock
+
+   Copyright (C) 2020 ????
```

Comment:
What should I write as copyright here? Xceed seems wrong.

Reply:
It might also be that this should be in the same file the [other extension methods](https://github.com/Dirkster99/AvalonDock/blob/master/source/Components/AvalonDock/Layout/Extensions.cs). Those methods work on interfaces in AvalonDock, these work on WPF classes.